cyences_app_for_splunk/default/transforms.conf

```diff
@@ -203,6 +203,21 @@ SOURCE_KEY = ObjectGUID
 REGEX = (.+)
 FORMAT = objectGUID::$1
 
+# MSSQL
+[mssql_audit_event]
+SOURCE_KEY = Message
+REGEX = Audit event:\s(?<mssql_audit_event>.*)
+
+[mssql_audit_fields]
```

Comment:
Do we need to extract the statement field explicitly?

Reply:
No. It will be extracted as part of this